### PR TITLE
chore: updated directories 2.0.2 to directories-next 1.0.2

### PR DESCRIPTION
**Pull request by drcpu-github**
Solves issue #1642.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -26,7 +26,7 @@ travis-ci = { repository = "https://github.com/witnet/witnet-rust", branch = "ma
 ansi_term = "0.12.1"
 bytecount = "0.6.0"
 ctrlc = "3.1.3"
-directories = "2.0.2"
+directories-next = "1.0.2"
```
Comment (tmpolaczyk) [inline Cargo.toml:29]: Actually, this line can be removed

```suggestion
```

Comment (drcpu-github) [reply Cargo.toml]: Yes, I was not sure why that dependency was included there since it wasn't used, but I did not want to break anything.

**Comment by drcpu-github** [comment]
Added suggestions and rebased master.

**Review by tmpolaczyk — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (tmpolaczyk) [reply Cargo.toml]: Yeah, ideally we would have some automated check that would detect unused dependencies, but we haven't found any.

**Pull request merged**